### DISCUSSION TITLE
Welcome to Portkey's Hacktoberfest 2024! 🎉

### DISCUSSION THREAD

**Discussion by ashirwad19** [Portkey-AI/gateway#659] [General]
# Welcome to Portkey's Hacktoberfest 2024! 🎉

We're excited to participate in Hacktoberfest and welcome contributors from around the world. Here's everything you need to know to get involved and make meaningful contributions to our project.

## How to Participate

1. **Sign up for Hacktoberfest**: Register at [hacktoberfest.com](https://hacktoberfest.com/).
2. **Find an issue**: Look for issues labeled with `hacktoberfest` in our repository.
3. **Fork the repo**: Create your own fork of the Portkey repository.
4. **Make your changes**: Work on the issue in your forked repo.
5. **Submit a PR**: Open a pull request from your fork to our main repository.

## Issue Tags

We've marked eligible issues with the following tags:
- `hacktoberfest`: All Hacktoberfest-eligible issues
- `good-first-issue`: Perfect for beginners
- `bug`: Help us squash bugs
- `enhancement`: Contribute to new features
- `documentation`: Improve our docs

## Rewards

- Complete 2 accepted PRs on the Portkey AI Gateway repo and receive a Portkey Swag Kit, including:
  - Portkey T-shirt
  - Stickers
  - Sipper
  - And much more!
- All accepted contributions will be acknowledged on our Contributors page.

## What We Expect from Contributors

- **Quality over quantity**: We value meaningful contributions that improve our project.
- **Follow our guidelines**: Please read our CONTRIBUTING.md file before starting.
- **Be respectful**: Adhere to our Code of Conduct in all interactions.
- **Engage in discussions**: Feel free to ask questions and participate in issue threads.

## Support Options

- **Discord**: Join our [Portkey Discord server](https://discord.gg/DD7vgKK299) for real-time support.
- **GitHub Discussions**: Post your questions in the Discussions tab of our repo.
- **Documentation**: Check out our [contributor guidelines](https://github.com/Portkey-AI/gateway/blob/b3ce2cc3d5e9e213c37fc19f25b38dddc54aaec9/.github/CONTRIBUTING.md#L4) for detailed information.

We're looking forward to your contributions. Let's make Portkey even better together! 💻🚀

Happy Hacking!

Team Portkey

**Comment by Ajithpmdwaraka** [comment]
Definitely
